Lib/test/datetimetester.py

```diff
@@ -1350,6 +1350,17 @@ def test_strftime(self):
         #check that this standard extension works
         t.strftime("%f")
 
+        # bpo-35066: make sure trailing '%' don't cause
+        # datetime's strftime to complain
+        failed = False
```

Comment:
Can you rename this to something more descriptive? The way it is now it's not clear what is failing. I would probably do it like this:

```python
time_supports_trailing_percent = True
try:
    _time.strftime('%')
except ValueError:
    time_supports_trailing_percent = False
```

Reply:
Ok, will do